<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloneWdgt</class>
 <widget class="QWidget" name="CloneWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>From:</string>
     </property>
     <property name="buddy">
      <cstring>txtUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="LineEdit" name="txtUrl">
     <property name="mandatory" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Alias:</string>
     </property>
     <property name="buddy">
      <cstring>txtRemoteAlias</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="LineEdit" name="txtRemoteAlias">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>origin</string>
     </property>
     <property name="mandatory" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>To:</string>
     </property>
     <property name="buddy">
      <cstring>txtPath</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="LineEdit" name="txtPath">
     <property name="mandatory" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="btnBrowseTo">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="chkAppendRepoName">
     <property name="text">
      <string>Append repository name</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>libMacGitverCore/Widgets/LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
